code/datums/mutations/sight.dm

```diff
@@ -18,6 +19,7 @@
 //Blind makes you blind. Who knew?
 /datum/mutation/human/blind
 	name = "Blindness"
+	desc = "Completely inhibits peripheral brain function."
```

Comment:
not particularly important, but i'd suggest making this about the eyes themselves since a brain issue would be the same as a brain trauma